### ISSUE TITLE
Switch from fabritone to baritone, since baritone 1.16.5 supports fabric

### ISSUE THREAD

**Issue by adrisj7** [toccatina/altoclef#36]
https://github.com/cabaletta/baritone/releases/tag/v1.6.3

Would keep baritone up to date, fix some issues and ALSO be a github fork, so it can be added to this organization for ease of access (very nice)

This seems to ONLY be 1.16.5, so this might not be the best route if we want to support earlier versions (like 1.16.1, or even 1.12)

**Comment by CodeF53** [comment]
The fabritone fork altoclef relies on doesn't support sodium.

Given a command baritone repeatedly says 
`[main/INFO]: [CHAT] [Baritone] Pausing since destination is at edge of loaded chunks`

**Comment by adrisj7** [comment]
Effectively done on the [ditch-fabritone](https://github.com/toccatina/altoclef/tree/ditch-fabritone) branch. I should close this only after I've merged, but we have duplicates of this so I guess I'll close this one for now.
